src/test/java/com/emc/object/s3/S3JerseyClientTest.java

```diff
@@ -1437,6 +1437,19 @@ public void testEmptyObjectChunked() {
         Assert.assertEquals("", client.readObject(getTestBucket(), key, String.class));
     }
 
+    @Test
+    public void testEmptyObjectFile() throws IOException {
+        String key = "empty-object";
+        int size = 0;
+        byte[] data = new byte[size];
+        new Random().nextBytes(data);
```

Comment:
Why creating data that will not be used later?

Reply:
Removed in new commit.